src/app/theming/angular/angular-sample.component.ts

```diff
@@ -15,34 +16,27 @@ export class AngularMaterialComponent implements OnInit {
     @ViewChild('dialog', { read: IgxDialogComponent, static: true })
     public dialog: IgxDialogComponent;
 
-    @ViewChild('dropdown', { read: IgxDropDownComponent, static: true })
-    public dropdown: IgxDropDownComponent;
-
-    @ViewChild(IgxOverlayOutletDirective, { static: true })
-    private igxOverlayOutlet: IgxOverlayOutletDirective;
-
-    @ViewChild(IgxOverlayOutletDirective, { static: true })
-    private igxOverlayOutlet1: IgxOverlayOutletDirective;
-
     @HostBinding('class')
     public themesClass = 'light';
 
     firstFormGroup: FormGroup;
     secondFormGroup: FormGroup;
     thirdFormGroup: FormGroup;
 
-    public themeOverlaySettings;
-    public dropDownSetting;
     private _dialogOverlaySettings2;
 
-    constructor(private _formBuilder: FormBuilder) {}
+    constructor(private _formBuilder: FormBuilder, private overlayContainer: OverlayContainer) {
+        overlayContainer.getContainerElement().classList.add('menu-theme');
+    }
 
     public lightTheme() {
         this.themesClass = 'light';
+        this.overlayContainer.getContainerElement().classList.remove('dark-menu-theme');
```

Comment:
Perhaps we can just have a method that resets the classes when the light/dark themes methods are called. In this way the `lightTheme` method will look more logical.

```suggestion
        this.toggleOverlayClass(this.themeClass);
```

Where `toggleOverlayClass` can look something like:

```ts
private toggleOverlayClasses(theme: 'light' | 'dark') {
    const overlayClasses = this.overlayContainer.getContainerElement().classList;

    switch (theme) {
        case 'light':
            overlayClasses.remove('dark-menu-theme');
            overlayClasses.add('light-menu-theme');
            break;
        case 'dark':
            overlayClasses.remove('light-menu-theme');
            overlayClasses.add('dark-menu-theme');
            break;
    }
}
```